src/MensaWidget/MWMenuBar.class.st

```diff
@@ -224,12 +279,80 @@ MWMenuBar >> target: anObject [
 ]
 
 {
-	#category : #dateManipulation,
-	#'squeak_changestamp' : 'Louis Christ 6/10/2024 17:59'
+	#category : #accessing,
+	#'squeak_changestamp' : 'Louis Christ 6/17/2024 15:20'
+}
+MWMenuBar >> time [
+
+	^ time
+]
+
+{
+	#category : #accessing,
+	#'squeak_changestamp' : 'Louis Christ 6/17/2024 15:20'
+}
+MWMenuBar >> time: anObject [
+
+	time := anObject
+]
+
+{
+	#category : #accessing,
+	#'squeak_changestamp' : 'Louis Christ 6/17/2024 15:20'
+}
+MWMenuBar >> timeToggle [
+
+	^ timeToggle
+]
+
+{
+	#category : #accessing,
+	#'squeak_changestamp' : 'Louis Christ 6/17/2024 15:20'
+}
+MWMenuBar >> timeToggle: anObject [
+
+	timeToggle := anObject
+]
+
+{
+	#category : #timeManipulation,
+	#'squeak_changestamp' : 'Louis Christ 6/17/2024 21:51'
+}
+MWMenuBar >> toggleTime [
+
+	self time = 'lunch' ifTrue:
+		[self time: 'dinner'] ifFalse:
```

Comment:
i think the ifTrue and the IfFalse should each move one line down, so it's immediately visible that we have two cases

Reply:
I like it as well we just need to align as a team how we format if statements